worlds/AutoWorld.py

```diff
@@ -91,7 +91,36 @@ class WebWorld:
 
     # Choose a theme for your /game/* pages
     # Available: dirt, grass, grassFlowers, ice, jungle, ocean, partyTime
-    theme = "grass"
+    theme: str = "grass"
+
+    # overridable method allowing you to create your own tracker. It needs to be designated as a class method to be picked
+    # up correctly. All the arguments available should give you the information you need to build one and the template html
+    # can also be overridden. This is a class method and the tracker is not typed to avoid circular importing.
+    def get_player_tracker(self, tracker_info):
```

Comment:
Let's get some typing info on tracker_info and typing info on the return of this function.

Reply:
to avoid circular importing I either have to move the PlayerTracker class to AutoWorld or BaseClasses and not give typing to its room variable (which isn't supposed to be edited by implementations anyways) or keep it how it currently is. That comment block above the method was outdated and I've since deleted it.

Reply:
Found a proper fix for this. In upcoming commit.